app-webpack/types/configuration/conf.d.ts

```diff
@@ -75,7 +75,7 @@ interface BaseQuasarConfiguration {
     remove: string[];
   };
   /** Add variables that you can use in index.template.html. */
-  htmlVariables?: { [index: string]: string };
+  htmlVariables?: Record<string, string | Record<string, unknown>>;
```

Comment:
Let's simply go with `any` as it's possible to use numbers, arrays, etc. as well.
```suggestion
  htmlVariables?: Record<string, any>;
```